index.css

```diff
@@ -0,0 +1,110 @@
+{box-sizing: border-box}
+
+
+
+body{
+
+}
+
+nav{
+    height: 120px;
+    width: 100%;
+    position: fixed;
+    top: 0;
+    z-index: 3;
+}
+
+nav img{
+    margin-left: 50px;
+}
+
+.navRight{
+    float: right;
+    margin-right: 200px;
+    padding-top: 30px;
+}
+
+
+.dropdown{
+    float: right;
+    margin-right: 200px;
+    margin-top: 70px;
+    display: none;
+    background-image: url("button.png");
+    background-size: 100%;
+    height: 5px;
+    width: 5px;
+    padding: 16px;
+    border: none;
+}
+
+nav li{
+    display: inline-block;
+    margin-top: 30px;
+}
+
+.liText{
+    font-family: "Arial Rounded MT Bold";
```

Comment:
if your going to use this across the whole website just put it as a style on the body. also put a websafe font as a backup